src/data/behandlerdialog/useReturLegeerklaring.tsx

```diff
@@ -0,0 +1,28 @@
+import { useValgtPersonident } from "@/hooks/useValgtBruker";
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+import { ISBEHANDLERDIALOG_ROOT } from "@/apiConstants";
+import { ReturLegeerklaringDTO } from "@/data/behandlerdialog/behandlerdialogTypes";
+import { post } from "@/api/axios";
+import { behandlerdialogQueryKeys } from "@/data/behandlerdialog/behandlerdialogQueryHooks";
+
+export const useReturLegeerklaring = (legeerklaringUuid: string) => {
+  const personident = useValgtPersonident();
+  const queryClient = useQueryClient();
+  const path = `${ISBEHANDLERDIALOG_ROOT}/melding/${legeerklaringUuid}/retur`;
+
+  const postReturLegeerklaring = async (
+    returLegeerklaring: ReturLegeerklaringDTO
+  ) => {
+    await post(path, returLegeerklaring, personident);
+  };
+
+  return useMutation({
+    mutationFn: postReturLegeerklaring,
+    onSuccess: () => {
+      // Returnerer Promise her slik at mutation er 'loading' til disse queryene er invalidert
+      return queryClient.invalidateQueries(
+        behandlerdialogQueryKeys.behandlerdialog(personident)
+      );
```

Comment:
Smart, er det noe vi ikke har pleid å gjøre?

Reply:
Joda, vi har gjort det et par andre steder også - typisk hvis man poster fra en modal og vil at data "bak" modalen skal være oppdatert når den lukkes. Kan for så vidt fjerne kommentaren her.